Tests/LibWeb/Text/input/css/singular-percentage-calc-crash.html

```diff
@@ -0,0 +1,12 @@
+<script src="../include.js"></script>
+<style>
+    div {
+        transform: translateX(calc(10%))
+    }
+</style>
+<div></div>
+<script>
+    test(() => {
+        println(`PASS! (Didn't crash)`);
+    });
+</script>
```

Comment:
This file doesn't end with a new line. I'm surprised the CI didn't pick it up.

Reply:
We don't worry about that for tests. Back in the day I think we manually removed the trailing newlines for layout tests just so they wouldn't add the extra text node.

Reply:
Oh ok, I'm always fighting the pre-commit hook when I import WPT tests that don't have one.

Reply:
That's odd, maybe your commit hook is outdated? I don't think I've had that issue but maybe I've been importing tests that don't lack the newline.